test/unit/scope-exec.test.js

```diff
@@ -0,0 +1,27 @@
+describe('scope module exec', function () {
+    var Scope = require('../../'),
+        scope;
+
+    beforeEach(function () {
+        scope = Scope.create();
+        scope.set('expect', expect);
+    });
+    afterEach(function () {
+        scope = null;
+    });
+
+    it('must be able to execute a simple script', function (done) {
+        scope.exec(`var a = 1 + 2;`, done); // eslint-disable-line quotes
```

Comment:
We used back ticks all over the tests since they are mostly multi-line and will run in node and be browserified for chrome. Only a few tests are single line. But for those, I am retaining convention since people tend to copy the simplest tests to build new ones.